homeassistant/config.py

```diff
@@ -87,19 +87,25 @@
 """
 
 
-def _valid_customize(value):
-    """Config validator for customize."""
-    if not isinstance(value, dict):
-        raise vol.Invalid('Expected dictionary')
+CUSTOMIZE_SCHEMA_ENTRY = vol.All(
```

Comment:
The only thing you need here, instead of all the other stuff is this:

```python
CUSTOMIZE_SCHEMA_ENTRY = vol.Schema({
  vol.Required(CONF_ENTITY_ID): vol.All(cv.ensure_list_csv, [cv.string])
}, extra=vol.ALLOW_EXTRA)
```